### DISCUSSION TITLE
阅读列表

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2954] [Developer]
# 重要提示

> ### 嗨，简悦全新的 [教程中心](https://kb.simpread.pro/) 已上线，**下面的内容已经已同步更新**，也可以 [访问这里](https://kb.simpread.pro/#/page/阅读列表) 查看。

***

> 手机端阅读的难题：碎片化阅读，因为 **功能越多，越没心思阅读**。阅读列表的设计理念：**给的越少，用的越专注**；希望通过此功能让你手机端的也拥有沉浸式阅读体验。

> 详细说明可以看 [这里](https://github.com/Kenshin/simpread/discussions/2954) 或 [这里](https://simpread.pro/api/#/reading)。

## 功能

* 支持 PWA，包括：[桌面端](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517974) · [Android](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517975) · [iOS](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517977) 系统
* [支持下拉刷新](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1518197)
* [暗色模式](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1518195)

## 特点

* 数据全部归用户所有，隐私权得到最大保证
* 通过简悦，可以让你的坚果云或 Dropbox 特定文件夹的内容在线查看

## 使用场景

- [内嵌到 Notion 中左侧查看，右侧笔记](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1518198)

## 如何配置

> 此功能依赖于 [开放平台](http://simpread.pro/developer)，如何开通 [请看这里](https://github.com/Kenshin/simpread/discussions/2757)，如何配置资料库 [请看这里](https://github.com/Kenshin/simpread/discussions/2956)。

# 关联

- [个人中心 #2959](https://github.com/Kenshin/simpread/discussions/2959)
- [收藏助手 #2921](https://github.com/Kenshin/simpread/discussions/2921)
- [免密码登录 #2974](https://github.com/Kenshin/simpread/discussions/2974)

**Comment by Kenshin** [comment]
# 桌面端

> 在 Windows 或 Mac 上使用 PWA 的方式，可配合 [收藏助手](https://github.com/Kenshin/simpread/discussions/2921) 一起使用。

https://user-images.githubusercontent.com/81074/138024539-a172e66e-66a3-42f6-b915-3e151b7458c3.mp4

## 如何安装

> 用基于 Chrome 内核的浏览器打开收藏助手后就能看到下图所示的提示

![image](https://user-images.githubusercontent.com/81074/138392588-5d1ed3a8-af9a-46de-9e34-264fb5731c6e.png)

> 也可以通过下图提示的方式进行安装

![image](https://user-images.githubusercontent.com/81074/138392778-9d265411-2fce-4360-aedf-cbb6fee88026.png)

### 桌面版效果

![image](https://user-images.githubusercontent.com/81074/138392885-fc0d942f-576a-48f0-8616-d5fcb2e52596.png)

### Mac 版支持 Dock 方式

![image](https://user-images.githubusercontent.com/81074/138392933-b5ee8de8-9bd0-482f-881e-c05a457db507.png)

**Reply by PrayerLi** [reply]
登录阅读列表的时候显示服务器发生错误怎么处理呀

**Reply by Kenshin** [reply]
解决方案 https://github.com/Kenshin/simpread/discussions/1567

另外，如果还是无法解决你的问题，请提 Issues https://github.com/Kenshin/simpread/issues/new

**Comment by Kenshin** [comment]
# Android

> 得益于 Chrome Android 可以使用 PWA，所以在 Android 设备上除了像桌面应用一样使用。

### 如何安装（步骤）

> 用浏览器打开阅读列表，注意：请使用 `https://simpread.pro/@id/reading/?token=<token>` 形式（可通过个人中心登录，即为此链接形式）

![Screenshot_2021-10-22-12-25-44-437_Chrome (1)](https://user-images.githubusercontent.com/81074/138395852-0dabf980-33c2-456f-a23f-fdcf7f05c6e4.png)

### 如何安装（视频）

https://user-images.githubusercontent.com/81074/138397336-d1212ae4-a918-45e6-bae6-624fd236a2e1.mp4

**Reply by starrycrown** [reply]
华为p30pro，用edge没动静，显示安装中后，然后就没动静了，最后使用的华为自带浏览器成功了。

**Reply by Kenshin** [reply]
Chrome 是可以的，但确实没有测试过 Edge

谢谢提示~

**Reply by StrollStars** [reply]
打开网页要输入授权码，这是啥？个人中心入口是什么？

**Reply by Kenshin** [reply]
@StrollStars

嗨，你可以看这个链接 https://www.yuque.com/kenshin/simpread/ieu1w6 的说明

**Comment by Kenshin** [comment]
# iOS

> 虽然 iOS 不完全支持 PWA，但仍会以「桌面 App」的形式安装和运行。

### 如何安装（步骤）

> 用浏览器打开阅读列表，注意：请使用 `https://simpread.pro/@id/reading/?token=<token>` 形式（可通过个人中心登录，即为此链接形式）

![阅读列表 iOS 安装-optimize](https://user-images.githubusercontent.com/81074/138397040-68dbc35e-1dff-4fd4-9d48-2ff7da5f4c0b.png)

### 如何使用（视频）

https://user-images.githubusercontent.com/81074/138397774-0d4f98ae-1987-4f82-b68c-916f191d8ca7.MP4

**Reply by felix2049lou** [reply]
刚试了在iOS确实可行，比去研究令手机访问http://localhost:7026/unread/ 相对容易一些。
补充细节：
登入开放平台後：
https://simpread.pro/developer
把其中的id复制
<img width="771" alt="Screenshot 2024-10-11 at 17 30 25" src="https://github.com/user-attachments/assets/2331ffb8-b86d-4319-80c5-bd1b1ddfe96a">
及其中的Token复制
<img width="741" alt="Screenshot 2024-10-11 at 17 24 32" src="https://github.com/user-attachments/assets/6aab5f0e-9606-4e10-ad6d-6c695be64f3e">

修改 https\://simpread.pro/@id/reading/?token=\<token\>的其体位置 : @你复制的id,把\<token\>改成你复制的token

同时把阅读列表设为密码访问：
<img width="807" alt="Screenshot 2024-10-11 at 17 32 31" src="https://github.com/user-attachments/assets/7d5324ac-5ec0-44f5-a7a3-aefdbde941d7">

**Reply by Kenshin** [reply]
一个提示：http://localhost:7026/unread/  与 `https://simpread.pro/@id/reading/?token=<token>` 是两种方式。

如果你已经成功配置后者，前者可以直接访问。😄

**Comment by Kenshin** [comment]
# 暗色模式

> 支持系统的暗色模式。

> 此行为支持：Web 端 · [桌面端](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517974) · [Android](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517975) · [iOS](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517977) 全平台

https://user-images.githubusercontent.com/81074/138406381-09c72eed-9a80-4862-b0cd-04d5aa7090a2.mp4

**Comment by Kenshin** [comment]
# 下拉刷新

> 此行为支持：[桌面端](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517974) · [Android](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517975) · [iOS](https://github.com/Kenshin/simpread/discussions/2954#discussioncomment-1517977) 全平台

https://user-images.githubusercontent.com/81074/138407112-134a80d3-6abd-4abe-b9aa-0dbb2ec0e7a3.mp4

**Comment by Kenshin** [comment]
# 阅读列表 +  Notion

> 将阅读列表 Embed 到 Notion 中，再配合 [收藏助手](https://github.com/Kenshin/simpread/discussions/2921)，完全可以实现在 iPad 等设备上的收藏 / 提炼功能。

> 注意：Embed 到 Notion 的链接为请使用 `https://simpread.pro/@id/reading/?token=<token>` 形式（可通过个人中心登录，即为此链接形式）

https://user-images.githubusercontent.com/81074/138407531-9ee28e76-c596-4f32-ba17-828f6437566b.mp4
